broker/src/main/java/org/apache/rocketmq/broker/longpolling/PullRequestHoldService.java

```diff
@@ -174,7 +174,15 @@ public void notifyMessageArriving(final String topic, final int queueId, final l
                     mpr.addPullRequest(replayList);
                 }
             }
+            else if (null == requestList) {
+                this.pullRequestTable.remove(key);
+            }
+           
+        }
+        else if (null == mpr) {
```

Comment:
pullRequestTable is ConcurrentHashMap, so null means this key is not existed, so this branch is redundant